nbextensions/bulkImportCell/main.js

```diff
@@ -18,19 +18,14 @@ define([
     'bluebird',
     'common/runtime',
     'common/error',
-    './bulkImportCell',
-    'kb_service/client/workspace',
-    'kb_service/utils',
-    'custom/custom'
```

Comment:
Don't need any of these - they should've been deleted along with the `get_workspace_info` call